codes/dart/chapter_computational_complexity/leetcode_two_sum.dart

```diff
@@ -0,0 +1,52 @@
+/**
+ * File: leetcode_two_sum.dart
+ * Created Time: 2023-2-11
+ * Author: Jefferson (JeffersonHuang77@gmail.com)
+ */
+
+import 'dart:collection';
+
+/* 方法一： 暴力枚举 */
+class SolutionBruteForce {
+  List<int> twoSum(List<int> nums, int target) {
+    int size = nums.length;
+    for (var i = 0; i < size - 1; i++) {
+      for (var j = i + 1; j < size; j++) {
+        if (nums[i] + nums[j] == target) return [i, j];
+      }
+    }
+    return [0];
+  }
+}
+
+/* 方法二： 辅助哈希表 */
+class SolutionHashMap {
```

Comment:
Remove the class if it is unnecessary. You can refer to the C++ code to see how it works.